### PR TITLE
Fix/form validation formats

### PR DESCRIPTION
**Pull request by vitydria**

### CODE REVIEW

File: src/components/WorkForm.js [hunk 1]
```diff
@@ -358,34 +436,39 @@ const WorkForm = () => {
           return value && value[0] && value[0].size <= 2097152
         }).test("type",t("workWithUs_workForm_schemaYup_curriculum3"),
           function (value) {
-            return value && value[0] && value[0].type === "application/pdf"
+            return value && value[0] && (
+              value[0].type === 'image/jpg' ||
```
Comment (davidadc) [inline src/components/WorkForm.js:440]:
```suggestion
              ['image/jpg', 'image/jpeg', 'application/pdf', 'application/msword'].includes(value[0].type)
```

A more readable way.

File: src/components/WorkForm.js [hunk 2]
```diff
@@ -358,34 +436,39 @@ const WorkForm = () => {
...
 
   const watchCurriculum = watch("curriculum");
- 
-
+  
   useEffect(() => {
-    if (watchCurriculum?.length > 0) {
+    if (watchCurriculum?.length > 0 &&
+      (watchCurriculum[0]?.type === 'image/jpg' || watchCurriculum[0]?.type === 'image/jpeg' ||
+      watchCurriculum[0]?.type === 'application/pdf' || watchCurriculum[0]?.type === 'application/msword')) {
```
Comment (davidadc) [inline src/components/WorkForm.js:469]: Same as previous validation.

**Review by davidadc — CHANGES_REQUESTED** [review]

File: src/components/WorkForm.js (hunk 1 above)
Comment (vitydria) [reply src/components/WorkForm.js:440]: I didn't know that way, gentleman

File: src/components/WorkForm.js (hunk 2 above)
Comment (vitydria) [reply src/components/WorkForm.js:469]: 🙏

Comment (vitydria) [reply src/components/WorkForm.js:469]: obrigado

**Review by davidadc — APPROVED** [review]

**Pull request merged**